src-plugins/medClut/medClutToolBox.cpp

```diff
@@ -116,84 +115,70 @@ medClutToolBox::~medClutToolBox(void)
 
 void medClutToolBox::updateView()
 {
-    medTabbedViewContainers * containers = this->getWorkspace()->stackedViewContainers();
-    QList<medViewContainer*> containersInTabSelected = containers->containersInTab(containers->currentIndex());
-    medAbstractView *view=NULL;
-    for(int i=0;i<containersInTabSelected.length();i++)
-    {
-        if (containersInTabSelected[i]->isSelected())
-        {
-            view = containersInTabSelected[i]->view();
-            break;
-        }
-    }
+    medAbstractView *view = this->getWorkspace()->stackedViewContainers()->getFirstSelectedContainerView();
 
-    if(!view)
+    if(view)
     {
-        return;
-    }
-
-    medAbstractLayeredView * medView = dynamic_cast<medAbstractLayeredView *> (view);
-    if ( !medView )
-        return;
-
-    if (medView==d->view)
-        return;
-
-    d->view = medView;
-
-    medAbstractData *data = d->view->layerData(0);
-    if (!data->identifier().contains( "vtkDataMesh" ))
-        return;
-    
-    medAbstractMeshData * dataMesh = static_cast<medAbstractMeshData*>(data);
-    vtkMetaDataSet * mesh = static_cast<vtkMetaDataSet*>(dataMesh->mesh());
-
-    if (!mesh || (mesh->GetType() != vtkMetaDataSet::VTK_META_SURFACE_MESH &&
-                         mesh->GetType() != vtkMetaDataSet::VTK_META_VOLUME_MESH))
-         return;
-
-    d->listOfAttributes.clear();
-    d->attributeBox->clear();
-
-    vtkImageView3D * view3d = static_cast<medVtkViewBackend*>(d->view->backend())->view3D;
+        medAbstractLayeredView * medView = dynamic_cast<medAbstractLayeredView *> (view);
 
-    if (view3d->GetDataSetCollection()->GetItem(0))
-    {
-        d->viewDataSet->insert(d->view,view3d->GetDataSetCollection()->GetItem(0)); 
-    }
-    else
-        return;
-    
-    for (int i = 0; i < mesh->GetDataSet()->GetPointData()->GetNumberOfArrays(); i++)
-    {
-        if (mesh->GetDataSet()->GetPointData()->GetArray(i)->GetNumberOfComponents() == 1)
+        if (medView!=d->view)
         {
-            d->listOfAttributes << QString::fromUtf8(mesh->GetDataSet()->GetPointData()->GetArrayName(i));
-            d->attributeBox->addItem("Point : "  + QString::fromUtf8(mesh->GetDataSet()->GetPointData()->GetArrayName(i)));
+            d->view = medView;
+
+            medAbstractData *data = d->view->layerData(0);
+            if (!data->identifier().contains( "vtkDataMesh" ))
+                return;
+
+            medAbstractMeshData * dataMesh = static_cast<medAbstractMeshData*>(data);
+            vtkMetaDataSet * mesh = static_cast<vtkMetaDataSet*>(dataMesh->mesh());
+
+            if (!mesh || (mesh->GetType() != vtkMetaDataSet::VTK_META_SURFACE_MESH &&
```

Comment:
Yes I know, but when I review I go through all the modified lines and it's not obvious that it's only indentation because you might have indented and changed something too. Then once I find a problem in the code, even if it was there before, I think it's normal to point it out. If you indent without changing anything, putting a comment to say what parts don't need review would really help reviewers.